static/js/components/TNSRobotsPage.jsx

```diff
@@ -1038,6 +1040,20 @@ const TNSRobotsPage = () => {
     );
   };
 
+  const renderAcknowledgments = (dataIndex) => {
+    const tnsrobot = tnsrobotList[dataIndex];
+    return (
+      <Tooltip
+        title={`Added at the end of the author list, e.g. 'First Last (University), ${
+          tnsrobot?.acknowledgments || "<insert_acknowledgments_here>"
```

Comment:
Won't this be confusing, to display the existing acknowledgements in the tooltip? Also, here and above, is the `<` expected as part of the string? If not, you may simply want to replace it with an example: "Jane Doe (CalTech), added spectroscopy" (or whatever makes sense).

Reply:
So the idea of that tooltip is explain a user what this "acknowledgments" field adds to the report. So if there is already a value, I felt like showing what the actual string send to TNS will look like was useful.

The "<insert_acknowledgments_here>" was just to make it clear that it's a placeholder where the acknowledgments will be.

I can change the format for sure, but the idea is just for the user to understand that it's basically a string appended to the end of the other list after a comma